src/app/shared/version-picker/version-picker.ts

```diff
@@ -13,10 +13,10 @@ const versionUrl = 'https://material.angular.io/assets/versions.json';
   templateUrl: './version-picker.html'
 })
 export class VersionPicker {
-  /** The currently running versin of Material. */
+  /** The currently running version of Material. */
   materialVersion = materialVersion;
   /** The possible versions of the doc site. */
-  docVersions = this.http.get(versionUrl);
+  docVersions = this.http.get<Array<VersionInfo>>(versionUrl);
```

Comment:
I've added types here, but this still fails in Ivy. I plan to address the Ivy failure in another branch.
```ts
version-picker.html(6,25): Type 'VersionInfo[] | null' is not assignable to type 'VersionInfo[] | Iterable<VersionInfo> | undefined'.
version-picker.html(8,8): Object is of type 'unknown'.
```

Reply:
AFAIK we usually prefer `VersionInfo[]` instead of `Array<VersionInfo>`.

Reply:
Updated.